.github/workflows/UploadDockerImage.yml

```diff
@@ -0,0 +1,43 @@
+name: Upload Docker Images
```

Comment:
nit: Can you add a comment at the top explaining what this does any why. Something like _"Pushes Docker images created from the deploy tool's templates to an internal ECR, so we can scan them for vulnerabilities."_

Reply:
Done